README.md

```diff
@@ -117,4 +117,5 @@ We aim to maintain compatibility with each corresponding Xline version, and upda
 
 | go-xline | Xline |
 | --- | --- |
-| v0.1.x | v0.6.x |
\ No newline at end of file
+| v0.1.0 | v0.6.0 |
+| v0.2.0 | v0.6.1 |
```

Comment:
Why is it `v0.2.0`? Does this release involve any new API?

Reply:
It was wrong, this release does not add new features, the correct version is `v0.1.1`.